indico/modules/events/abstracts/models/email_logs.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from sqlalchemy.dialects.postgresql import ARRAY, JSON
+
+from indico.core.db.sqlalchemy import db, UTCDateTime
+from indico.util.date_time import now_utc
+from indico.util.string import format_repr, return_ascii
+
+
+class AbstractEmailLogEntry(db.Model):
+    __tablename__ = 'email_logs'
+    __table_args__ = {'schema': 'event_abstracts'}
+
+    id = db.Column(
+        db.Integer,
+        primary_key=True
+    )
+    abstract_id = db.Column(
+        db.Integer,
+        db.ForeignKey('event_abstracts.abstracts.id'),
+        index=True,
+        nullable=False
+    )
+    email_template_id = db.Column(
+        db.Integer,
+        db.ForeignKey('event_abstracts.email_templates.id'),
+        index=True,
+        nullable=True
+    )
+    sent_dt = db.Column(
+        UTCDateTime,
+        nullable=False,
+        default=now_utc
+    )
+    recipients = db.Column(
+        ARRAY(db.String),
+        nullable=False
+    )
+    subject = db.Column(
+        db.String,
+        nullable=True,
```

Comment:
Technically yes, but do we allow it? Hell no! Nothing worse than emails with no subject or just a plain `Re:` without anything after it.